examples/CMakeLists.txt

```diff
@@ -22,6 +22,10 @@ endif()
 append(CMAKE_C_FLAGS "${CMAKE_EXAMPLE_CCXX_FLAGS}")
 append(CMAKE_CXX_FLAGS "${CMAKE_EXAMPLE_CCXX_FLAGS}")
 
+if(DNNL_SYCL_CUDA)
+    append(CMAKE_CXX_FLAGS "-fsycl-targets=nvptx64-nvidia-cuda")
```

Comment:
This flag is needed to tell the sycl compiler which device to build kernels for. The examples `gpu-sycl-interop-buffer-cpp` and `gpu-sycl-interop-usm-cpp` use non-host task SYCL kernels which require knowledge of which devices you want to run. You don't need to set this flag when building the library as all kernels are host-tasks so they do not need to be compiled to the device.
The warning is to be expected, it happens when linking the cuda builtins.

Reply:
I see, makes sense, thanks for pointing this out.
Can you please also add `append(CMAKE_CXX_FLAGS "-Wno-linker-warnings")` because we have `-Werror` enabled by default and this warning breaks the build.

Reply:
Done